db/schema.rb

```diff
@@ -106,6 +106,8 @@
     t.text "original_cc_ship_to_number"
     t.datetime "cc_import_api_timestamp"
     t.string "cc_import_api_transaction_id"
+    t.boolean "cc_rb_user"
+    t.boolean "original_cc_rb_user"
```

Comment:
change was introduced elsewhere can we delete these lines please

Reply:
Good spot